web/tests/functional/config/__init__.py

```diff
@@ -0,0 +1,81 @@
+# coding=utf-8
```

Comment:
We have a lot of config files and values, what do you think about renaming the test directory to `cli_config` or something different which indicates that these are the tests for the cli configuration?